web3.js/src/connection.ts

```diff
@@ -2120,6 +2120,18 @@ export type ConnectionConfig = {
   confirmTransactionInitialTimeout?: number;
 };
 
+function warnListenerRemovedWithoutActiveSubscription(
```

Comment:
nit: rather than checking what this method actually does, it would be nice if the call-site looked like this: `console.warn(createSubscriptionWarningMessage(id, label));`